src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -406,242 +406,6 @@ def pair_wise_sigmoid_focal_loss(inputs: Tensor, labels: Tensor, alpha: float =
     return loss / height_and_width
 
 
-# Copied from transformers.models.detr.modeling_detr.DetrAttention
-class DetrAttention(nn.Module):
-    """
-    Multi-headed attention from 'Attention Is All You Need' paper.
-
-    Here, we add position embeddings to the queries and keys (as explained in the DETR paper).
-    """
-
-    def __init__(
-        self,
-        embed_dim: int,
-        num_heads: int,
-        dropout: float = 0.0,
-        bias: bool = True,
-    ):
```

Comment:
hmm.. not sure why this code is removed, can you clarify, please?

Reply:
It's a bit unusual, but in `transfomers` we do not do cross import between models (exception are the "modular_*" ones). So instead modules should be included with "# Copied from" statement on top, and then they can be synced across models with `make fix-copies` command

Reply:
Please revert it to the initial state and run the `make fix-copies` to sync make CI happy 😄

Reply:
> But won't that be redundant? I mean in this case we are just redefining the exact same code in this class whereas it can be imported easily?

Yes, there are some advantages and disadvantages of such an approach. The idea is to make models as independent as possible + make code explorable inside the single file, so you don't need to jump across different modules in different locations to understand the code.

Reply:
I understand. Thanks for clarifying.